adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -532,13 +611,61 @@ func createRoute(params *routeCreateParams) *routev3.Route {
 		Metadata:  nil,
 		Decorator: decorator,
 		TypedPerFilterConfig: map[string]*any.Any{
-			wellknown.HTTPExternalAuthorization: filter,
+			wellknown.HTTPExternalAuthorization: extAuthzFilter,
+			wellknown.Lua:                       luaFilter,
 		},
 		ResponseHeadersToRemove: responseHeadersToRemove,
 	}
 	return &router
 }
 
+func getInlineLuaScript(requestInterceptorURL string, responseInterceptorURL string) string {
+	requestI := ""
+	if len(requestInterceptorURL) > 0 {
+		requestI = "function envoy_on_request(request_handle)" +
+			"\n local headers, response = request_handle:httpCall(" +
+			"\n     \"" + requestInterceptorURL + "\"," +
+			"\n     {" +
+			"\n       [\":method\"] = \"GET\"," +
+			"\n       [\":path\"] = \"/\"," +
+			"\n       [\":authority\"] = \"routing\"," +
+			"\n     }," +
+			"\n     \"hello from lua request\"," +
+			"\n     5000" +
+			"\n   )" +
+			"\n request_handle:logInfo(\"Hello Amali from router.\")" +
+			"\n request_handle:headers():add(\"amali-req\", \"hello\")" +
+			"\nend" +
+			"\n"
```

Comment:
what if we use raw string within backquotes?
or read from a file and format it with URLs